src/options.rs

```diff
@@ -78,8 +81,8 @@ pub struct BuildOptions {
     #[structopt(
         short = "s",
         long = "sanitizer",
-        possible_values(&["address", "leak", "memory", "thread"]),
-        default_value = "address",
+        possible_values(&["address", "leak", "memory", "thread", "none"]),
+        default_value = "none",
```

Comment:
I'm leaning towards leaving the default value as "address" for compatibility and for not silently moving folks who were relying on asan for their unsafe code away from this without them knowing.

@Manishearth do you have thoughts on this?

-------------------------

Either way we go with this we need a new test. If we keep the existing default, then let's add a test for using `-s none`. Otherwise, if we change the default to no sanitizer, then we need a test for enabling asan with `-s address`.

Reply:
Yes, would prefer to keep the default here.

Reply:
Done, plus cloned the panic test without a sanitizer to have coverage for failure without any sanitizers.